OpenEdXMobile/res/layout/reset_dialog_email_input.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:baselineAligned="false"
```

Comment:
`wrap_content` means the layout will wrap to the largest width and height amongst it's children. The other one (probably the progress spinner) should still be centered. Baseline alignment should not even be applicable.